roadmap.md

```diff
@@ -99,7 +99,7 @@ Architecture:
 - [.NET Standard](https://github.com/dotnet/standard)
 - [.NET Core Tools/CLI](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/index)
 
-Version 2.0 minumum OS Support (Proposed):
+Version 2.0 minimum OS Support (Proposed):
 
 OS                            |Version                        |Architectures|Configurations|Notes
```

Comment:
I was trying to intentionally just list the minimum for any given family. Do you think it should be version ranges? Fedora 26 would also be in that list.